src/Semantic/Task.hs

```diff
@@ -269,6 +269,12 @@ runParser blob@Blob{..} parser = case parser of
       parseToAST (configTreeSitterParseTimeout config) language blob
         >>= maybeM (throwError (SomeException ParserTimedOut))
 
+  UnmarshalParser language ->
+    time "parse.tree_sitter_ast_parse" languageTag $ do
```

Comment:
I’ve gone with the same timing key, but I wonder if we should actually use a new one since it’s doing both the `tree-sitter` parse & unmarshalling? cc @tclem for thoughts.